packages/amplify-graphql-index-transformer/src/graphql-primary-key-transformer.ts

```diff
@@ -116,10 +119,8 @@ function validate(config: PrimaryKeyDirectiveConfiguration, ctx: TransformerCont
   const { object, field, sortKeyFields } = config;
 
   validateNotSelfReferencing(config);
-
-  const modelDirective = object.directives!.find(directive => {
-    return directive.name.value === 'model';
-  });
+  validateCPKFeatureFlag(ctx,config);
```

Comment:
If there are 2 models with primary keys, can you confirm that we don't throw the warning twice?

Reply:
Thank you Phani - it was warning 2 times when my schema has 2 @primarykey.
So, changed my code and now the warning isn't producing twice